src/Pie.js

```diff
@@ -59,11 +59,13 @@ const Pie = ({ series, colors, radius, innerRadius, backgroundColor,strokeCap})
           strokeWidth={width}
         />
         <RingShape radius={radius} width={width} color={backgroundColor} />
-        {series.map((item, idx) => {
+        {sections.map((section, idx) => {
+          const { percentage, color } = section;
```

Comment:
I think we could use `const { percentage, color, ...rest } = section;` to address your request in #26

Reply:
I was thinking to make a separate PR for testID support if that's alright